docs/src/main/asciidoc/config.adoc

```diff
@@ -444,31 +444,31 @@ Using `@ConfigProperty("quarkus.profile")` will *not* work properly.
 
 === Using placeholders
 
-Quarkus supports the use of placeholders in the `application.properties` file on build time.
+Quarkus supports the use of placeholders in the `application.properties` file at build time.
```

Comment:
@dmlloyd could you have a look at this new section?

I would have expected this to be also valid at runtime but maybe I'm too optimistic?

Reply:
Expressions work at the time that the configuration property is read.  If it's a build time property, the expression is expanded at build time, but if it's a run time property, the expression is preserved at build time and expanded at run time.

Also I'm not sure we should be using the term "placeholder"; the term we've used in MP Config so far is "expression" or "property expression".

Reply:
Yeah I didn't like the term placeholder either.

OK so we need some further tuning of this section.

Reply:
:-) I used that term translating the one in German that I had in mind when writing it. Will be more careful/inspired next time.